requirements.txt

```diff
@@ -1,6 +1,6 @@
 lxml>=3.2.4
 # ncclient version 0.6.10 has issues with PyEZ(junos-eznc) and needs to be avoided
-ncclient==0.6.13
+ncclient==0.6.15
```

Comment:
I think we should make this `ncclient>=0.6.15` (to avoid pinning to one specific ncclient release)